src/components/themed/FilledTextInput.tsx

```diff
@@ -215,10 +215,11 @@ export const FilledTextInput = React.forwardRef<FilledTextInputRef, FilledTextIn
 
   const InputComponent = numeric ? StyledNumericInput : StyledAnimatedTextInput
 
+  const { keyboardType, returnKeyType } = props
```

Comment:
This should move to the top of the file, were we destructure all the other props.